### PR TITLE
Suggested changes to TAP 11 and POUF 1

### PR DESCRIPTION
**Pull request by joshuagl**
I was reading about POUF's in TAP 11, which made me want to update POUF 1 to match the current state of the reference implementation. Which further made me want to suggest some changes to TAP 11 to track what I think are useful pieces of information in the POUF.

Changes in the PR include:
* recommend the POUF list version numbers of the implementation which are covered by the POUF
* suggest POUF's include a list of changes across versions
* Update POUF 1 to match the reference implementation, including the TAP 11 suggestions above

### CODE REVIEW

File: tap11.md [hunk 1]
```diff
@@ -95,12 +95,13 @@ At a minimum, a POUF shall contain the following sections:
   * Title:
   * Version:
   * Last-Modified:
-  * Author: optional list of authors' real names and email addrs
+  * Author: optional list of authors' real names and email addresses
   * Status: Draft / In Use / Obsolete
   * TUF Version Implemented:
```
[the remark this answers is not in the document]
Comment (joshuagl) [reply tap11.md:100]: That makes sense, a POUF version may cover more than one spec version.

**Comment by joshuagl** [comment]
> Thanks for updating these. I just have a couple of additional clarifications that could be added.

Thanks for the quick review and the suggested clarifications. I've used GitHub's UI to commit those suggestions. Please let me know if you would rather I do some git rebase & fixups to integrate them into the patch series as modifications to the existing patches, rather than as additional commits.

**Comment by mnm678** [comment]
There are just a couple, so I think the UI commits should be fine.

**Comment by joshuagl** [comment]
Thanks for the review and approval @mnm678. What's the process for merging changes in this repo?

**Comment by mnm678** [comment]
I think the pr just needs at least one approving review. I don't have write access to this repo though, so maybe @JustinCappos or @trishankatdatadog can confirm.

**Comment by JustinCappos** [comment]
@lukpueh might be better to look at this because he has fresher eyes on this topic.  I have a hard time seeing it as an outsider would...

**Comment by joshuagl** [comment]
I've pushed an extra patch to mark TAP 11 as Accepted, as discussed in the editors meeting on Friday

**Comment by JustinCappos** [comment]
> I've pushed an extra patch to mark TAP 11 as Accepted, as discussed in the editors meeting on Friday

I think we also need the README.md updated as part of this change.  Once that's done unless someone else objects, I will merge the PR...

**Comment by JustinCappos** [comment]
Great!  Will merge!

**Pull request merged**